README.md

```diff
@@ -200,7 +200,7 @@ Note: Although RetDec now supports a system-wide installation ([#94](https://git
   * `cd retdec`
   * `mkdir build && cd build`
   * `cmake .. -DCMAKE_INSTALL_PREFIX=<path>`
-  * `make -jN` (`N` is the number of CPU cores to use for parallel build)
+  * `make -jN` (`N` is the number of  of processes to use for parallel build, typically number of cores + 1 gives fastest compilation time)
```

Comment:
In all cases, there is `of  of processes` (double `of` and double space) instead of `of processes`.